src/main/java/org/acra/sender/EmailIntentSender.java

```diff
@@ -66,8 +66,12 @@ private String buildBody(@NonNull CrashReportData errorContent) {
 
         final StringBuilder builder = new StringBuilder();
         for (ReportField field : fields) {
+            String value = errorContent.get(field);
+            if (value != null) {

```

Comment:
I don't believe value can ever be null. If it is listed as a field then it should always have a value. And the previous code would have broken on line 74 with an NullPointerException.